includes/classes/blocks/class-bindings.php

```
@@ -416,6 +428,14 @@ public function build_itinerary_field( $build = '', $field = '', $count = 1 ) {
 		return $build;
 	}
 
+	public function maybe_get_prefix( $key, $build ) {
+		$pattern = '/<p\s+[^>]*\bclass="[^"]*\b' . esc_attr( $key ) . '\b[^"]*"[^>]*>(.*?)<\/p>/is';
+		if ( preg_match( $pattern, $build, $matches ) ) {
+			return str_replace( 'Card Link', '', $matches[1] );
+		}
+		return '';
+	}
```

PHPMD (2.15.0)

The method maybe_get_prefix is not named in camelCase. (undefined)

(CamelCaseMethodName)